addons/editor-extra-keys/addon.json

```
@@ -11,12 +11,12 @@
   "info": [
     {
       "type": "warning",
-      "text": "The \"experimental keys\" include equals, slash, semicolon and more. They may not work on all operating systems or keyboard layouts.",
+      "text": "The \"experimental keys\" include keys such as equals, backslash, and semicolon. They may not work on all operating systems or keyboard layouts.",
       "id": "experimentalKeysWarn"
     },
     {
       "type": "warning",
-      "text": "The \"Shift keys\" include keys which typically require the Shift key and a number key, like hashtag, exclamation mark and more. These keys only work with the \"when () key pressed\" block and do not work on all operating systems or keyboard layouts.",
+      "text": "The \"Shift keys\" include keys which typically require the Shift key and a number key, like hashtag (pound) and exclamation point. These keys only work with the \"when () key pressed\" block and do not work on all operating systems or keyboard layouts.",
```

```suggestion
      "text": "The \"Shift keys\" include keys which typically require the Shift key and a number key, like hashtag and exclamation point. These keys only work with the \"when () key pressed\" block and do not work on all operating systems or keyboard layouts.",
```
"Pound" is often used to refer to `£` sign.